TheWarWithin/RogueOutlaw.lua

```diff
@@ -1382,22 +1501,18 @@ spec:RegisterAbilities( {
 
         startsCombat = true,
         texture = 136189,
+        
 
         cp_gain = function () return 1 + ( buff.broadside.up and 1 or 0 ) end,
 
         handler = function ()
             gain( action.sinister_strike.cp_gain, "combo_points" )
             removeStack( "snake_eyes" )
 
-            if talent.unseen_blade.enabled and debuff.unseen_blade.down then
```

Comment:
Does simc track this using a fake `unseen_blade` aura with stacks?  If so, we should replicate so the importer works.